test/test.js

```diff
@@ -8,6 +8,18 @@ var Runner = require('../lib/runner');
 var fork = require('../lib/fork');
 var ava = require('../lib/test');
 
+// global Promise for zen-observable
+if (!global.Promise) {
+	Object.defineProperty(global, 'Promise', {
+		value: Promise,
+		configurable: true,
+		enumerable: false,
+		writable: true
+	});
+}
+
+var Observable = require('zen-observable');
+
```

Comment:
It's in test/test.js for Observables only as we want a lightweight implementation of the observable spec to test with. `zen-observable` "[Requires ES6 Promises or a Promise polyfill](https://github.com/zenparsing/zen-observable#zen-observable)" and this looked like a simple way to do that without also having a Symbol polyfil at the same time.

Maybe @blesh or @zenparsing have a suggestion?